### PR TITLE
Docs: Correct time forecast for tests.

### PR DESCRIPTION
**Pull request by jrpool**
**What is the purpose of this pull request? (put an "X" next to item)**

[X] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

**What changes did you make? (Give an overview)**
Amended forecast of time tests will take from a few seconds to 2 minutes, to reflect reality.

**Is there anything you'd like reviewers to focus on?**

### CODE REVIEW

**Comment by kaicataldo** [comment]
CI failure not related to this PR. Restarting.

File: docs/developer-guide/development-environment.md [hunk 1]
```diff
@@ -55,9 +55,7 @@ Running the tests is the best way to ensure you have correctly set up your devel
 npm test
 ```
 
-The testing takes a few seconds to complete. If any tests fail, that likely means one or more parts of the environment setup didn't complete correctly. The upstream tests always pass.
-
-
+The testing takes about 2 minutes to complete. If any tests fail, that likely means one or more parts of the environment setup didn't complete correctly. The upstream tests always pass.
```
Comment (kaicataldo) [inline docs/developer-guide/development-environment.md]: Can we maybe change this to `a few minutes to complete` instead? That way we don't have to keep updating this should they change in the future!

Comment (jrpool) [reply docs/developer-guide/development-environment.md]: Sure. Good idea. Will do.

**Comment by kaicataldo** [comment]
Thanks for contributing to ESLint!

**Pull request merged**